src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+
+export const Person = (props) => {
+  const { name, age, sex, isMarried, partnerName } = props.person;
+
+  let partner;
+  if (sex === 'm' && isMarried) {
+    partner = 'wife';
+  } else {
+    partner = 'husband';
+  }
```

Comment:
```suggestion
  const  partner = sex === 'm' ? 'wife' : 'husband';
  const personPartner = isMarried ? `${partnerName} is my ${partner}` : 'I am not married';
```
no need to check on isMarried for partner const.
alreade exists such check for final text.